css/style.css

```diff
@@ -0,0 +1,208 @@
+/* Downloaded Fonts */
+/* font-family: 'Anton', sans-serif;
+font-family: 'Hind Siliguri', sans-serif;
+font-family: 'Exo', sans-serif;
+font-family: 'Cinzel', serif;
+font-family: 'Pathway Gothic One', sans-serif;
+font-family: 'K2D', sans-serif; */
+
+html { 
+  background: url("../images/Skola.jpg") no-repeat center center fixed;
+  background-size: cover;
+}
+
+body {
+  margin: 0;
+}
+
+.page-container{
+  position: relative;
+  min-height: 100vh;
+}
+
+.content-wrap{
+  padding-bottom: 15px;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: end;
+  padding-top: 10px;
+  padding-right: 10px;
+}
+
+li {
+  display: inline;
+}
+
+a:hover {
+  color: #434343;
+}
+
+a {
+  color: whitesmoke;
+  font-family: 'K2D', sans-serif;
+  text-decoration: none;
+  margin: 5px;
+}
+
+/* INDEX PAGE */
```

Comment:
You should have different files for different page's css files.

Reply:
I will do this moving forward with my projects for this example I am keeping them the same.